docs/standard/glossary.md

```diff
@@ -286,6 +290,10 @@ Universal Windows Platform.
 
 An implementation of .NET that is used for building modern, touch-enabled Windows applications and software for the Internet of Things (IoT). It's designed to unify the different types of devices that you may want to target, including PCs, tablets, phones, and even the Xbox. UWP provides many services, such as a centralized app store, an execution environment (AppContainer), and a set of Windows APIs to use instead of Win32 (WinRT). Apps can be written in C++, C#, Visual Basic, and JavaScript. When using C# and Visual Basic, the .NET APIs are provided by .NET 5 (and .NET Core) and later versions.
 
+## workload
+
+A type of app someone is building. More generic than [app model](#app-model). Examples are Mobile, Desktop, IoT, Web, and Games.
```

Comment:
Maybe give examples of how they're used in context?

Reply:
@gewarren Does the added sample sentence help?

Reply:
I'm not sure, because it doesn't actually use the word "workload". Is the definition more about a concept than an actual term that's used in the docs?